src/http/resource.ts

```diff
@@ -9,6 +9,11 @@ import * as Drash from "../../mod.ts";
  *     https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Identifying_resources_on_the_Web
  */
 export class Resource implements Drash.Interfaces.IResource {
+  /**
+   * Internal property used to identify this as a Drash resource.
+   */
+  protected drash_resource = true;
```

Comment:
@Guergeiro @ebebbington,  it was a bit difficult for me to identify an extended resource as a `Drash.Resource`, so i added `drash_resource` to the base resource class. not sure if this is the best approach, but it's what this PR is currently working with. thoughts on a better solution?

Reply:
Works for me.